service.go

```diff
@@ -37,6 +37,17 @@ func SetupLogging(name, version string) error {
 		"ver":  version,
 		"host": hostname,
 	}
+
+	for _, m := range fields {
+		for k, v := range m {
+			if k == "srv" || k == "ver" || k == "host" {
```

Comment:
would it make sense to use constants for these ?

Reply:
done